test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -382,14 +240,20 @@ public void testFilepathColumnModeServesResult() throws Exception {
         Response.class.getClassLoader(),
         new Class[] { Response.class }, far);
     Map<String, String> cfg = new TreeMap<String, String>();
-    cfg.put("columnName", "my-filepath-col");
+    cfg.put("columnName", "filepath");
     File testFile = null;
     try {
```

Comment:
Yes, I planned to do that in a followup CL. I wanted to keep this focused on the H2 change.